### ISSUE TITLE
Broken Scons file refers to an undefined dictionary key 'bits'

### ISSUE THREAD

**Issue by kilon** [Zylann/godot_voxel#434]
```
scons: Reading SConscript files ...
Building for platform "linuxbsd", architecture "x86_64", editor, target "release_debug".
KeyError: 'bits':
  File "/home/kilon/coding/kyklops/SConstruct", line 851:
    SConscript("modules/SCsub")
  File "/usr/lib/python3.10/site-packages/SCons/Script/SConscript.py", line 660:
    return method(*args, **kw)
  File "/usr/lib/python3.10/site-packages/SCons/Script/SConscript.py", line 597:
    return _SConscript(self.fs, *files, **subst_kw)
  File "/usr/lib/python3.10/site-packages/SCons/Script/SConscript.py", line 285:
    exec(compile(scriptdata, scriptname, 'exec'), call_stack[-1].globals)
  File "/home/kilon/coding/kyklops/modules/SCsub", line 34:
    SConscript(base_path + "/SCsub")
  File "/usr/lib/python3.10/site-packages/SCons/Script/SConscript.py", line 660:
    return method(*args, **kw)
  File "/usr/lib/python3.10/site-packages/SCons/Script/SConscript.py", line 597:
    return _SConscript(self.fs, *files, **subst_kw)
  File "/usr/lib/python3.10/site-packages/SCons/Script/SConscript.py", line 285:
    exec(compile(scriptdata, scriptname, 'exec'), call_stack[-1].globals)
  File "/home/kilon/coding/kyklops/modules/voxel/SCsub", line 110:
    SConscript('thirdparty/fast_noise_2/SConscript', exports = ["env", "env_voxel"])
  File "/usr/lib/python3.10/site-packages/SCons/Script/SConscript.py", line 660:
    return method(*args, **kw)
  File "/usr/lib/python3.10/site-packages/SCons/Script/SConscript.py", line 597:
    return _SConscript(self.fs, *files, **subst_kw)
  File "/usr/lib/python3.10/site-packages/SCons/Script/SConscript.py", line 285:
    exec(compile(scriptdata, scriptname, 'exec'), call_stack[-1].globals)
  File "/home/kilon/coding/kyklops/modules/voxel/thirdparty/fast_noise_2/SConscript", line 90:
    if env["bits"] == "32":
  File "/usr/lib/python3.10/site-packages/SCons/Environment.py", line 387:
    return self._dict[key]
mv: cannot stat './bin/godot.linuxbsd.opt.tools.64': No such file or directory
```
Built using 
scons -j20 platform=linuxbsd target=release_debug bits=64

**Comment by Zylann** [comment]
The `env["bits"]` variables were removed yesterday by https://github.com/godotengine/godot/pull/55778.
I need to figure out what I need to replace this with.

**Comment by Zylann** [comment]
I fixed the `bits` in bf10c137ac37419df5e342ee4719beea8a99a6ad.

**Comment by kilon** [comment]
everything builds fine down, well done, closing this issue now
